### ISSUE TITLE
Sporadic test failure (3x): SpringTemplateJMS1Test

### ISSUE THREAD

**Issue by jkwatson** [open-telemetry/opentelemetry-java-instrumentation#528]
```
Condition failed with Exception:

receivedMessage.text == "responded!"
|               |
null            java.lang.NullPointerException: Cannot get property 'text' on null object
                	at SpringTemplateJMS1Test.send and receive message generates spans(SpringTemplateJMS1Test.groovy:102)

	at SpringTemplateJMS1Test.send and receive message generates spans(SpringTemplateJMS1Test.groovy:102)
Caused by: java.lang.NullPointerException: Cannot get property 'text' on null object
	... 1 more
```

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1341/workflows/69eb47e6-768d-46b9-bd9a-2ce73f86d440/jobs/10878/tests

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/1349/workflows/706a5c59-70a1-40cf-8638-910db03c3f8b/jobs/10950/tests

**Comment by trask** [comment]
Closing per #925

**Comment by trask** [comment]
This one is still happening:

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-java-instrumentation/2927/workflows/6ad8a3bc-80da-4a1b-8b77-431822668102/jobs/16849

**Comment by iNikem** [comment]
Closed by #1327
